cm.py

```diff
@@ -145,7 +145,7 @@ def main():
   # get list
   else:
     for report in API_DCM( config, auth, iterate=True).reports().list(**kwargs).execute():
-      print(json.dumps(report, indent=2, sort_keys=True))
+      print(yaml.dump(report, indent=2, sort_keys=True))
```

Comment:
Again, the API is in JSON:
https://developers.google.com/doubleclick-advertisers/rest/v4/reports